examples/identity/identity.bal

```diff
@@ -5,17 +5,17 @@ class MyClass {
 public function main() {
     MyClass obj1 = new MyClass();
     MyClass obj2 = new MyClass();
-    // `b1` will be true.
+    // The result will be true.
     boolean _ = (obj1 === obj1);
```

Comment:
Instead of saying `The result will be false.` Maybe we can assign a variable as previously, say b1, and then io:println() all of them at the end? This will also make sure the bbe output is correct.

Reply:
+1, we do have "run in playground" button in BBEs. So if we are not printing anything, users wont see anything.